.devcontainer/devcontainer.json

```
@@ -0,0 +1,26 @@
+// https://aka.ms/devcontainer.json
+{
+	"name": "Laravel Sail",
+	"dockerComposeFile": [
+		"../docker-compose.yml"
+	],
+	"service": "laravel.test",
+	"workspaceFolder": "/var/www/html",
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				// "mikestead.dotenv",
+				// "amiralizadeh9480.laravel-extra-intellisense",
+				// "ryannaddy.laravel-artisan",
+				// "onecentlin.laravel5-snippets",
+				// "onecentlin.laravel-blade"
+			],
+			"settings": {}
+		}
+	},
+	"remoteUser": "sail",
+	"postCreateCommand": "chown -R 1000:1000 /var/www/html && echo 'alias sail=\"[ -f sail ] && sh sail || sh vendor/bin/sail\"' >> ~/.bashrc"
```

```suggestion
	"postCreateCommand": "chown -R 1000:1000 /var/www/html"
```

I would suggest to remove sail alias here